docs/tutorials/reporting.md

```diff
@@ -33,188 +35,97 @@ By default, the timestamp format of the output files is ISO 8601. This can be ch
-            "Other": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/RDS/rds-auto-minor-version-upgrade.html"
-        },
-        "Recommendation": {
-            "Text": "Enable auto minor version upgrade for all databases and environments.",
-            "Url": "https://aws.amazon.com/blogs/database/best-practices-for-upgrading-amazon-rds-to-major-and-minor-versions-of-postgresql/"
-        }
-    },
-    "Categories": [],
-    "Notes": "",
-    "Compliance": {
-        "CIS-1.4": [
-            "1.20"
-        ],
-        "CIS-1.5": [
-            "1.20"
-        ]
-    }
-},
-{
-    "AssessmentStartTime": "2022-12-01T14:16:57.354413",
-    "FindingUniqueId": "",
-    "Provider": "aws",
-    "Profile": "dev",
-    "AccountId": "ACCOUNT_ID",
-    "OrganizationsInfo": null,
-    "Region": "eu-west-1",
-    "CheckID": "rds_instance_minor_version_upgrade_enabled",
-    "CheckTitle": "Ensure RDS instances have minor version upgrade enabled.",
-    "CheckType": [],
-    "ServiceName": "rds",
-    "SubServiceName": "",
-    "Status": "PASS",
-    "StatusExtended": "RDS Instance rds-instance-id has minor version upgrade enabled.",
-    "Severity": "low",
-    "ResourceId": "rds-instance-id",
-    "ResourceArn": "",
-    "ResourceType": "AwsRdsDbInstance",
-    "ResourceTags": {},
-    "Description": "Ensure RDS instances have minor version upgrade enabled.",
-    "Risk": "Auto Minor Version Upgrade is a feature that you can enable to have your database automatically upgraded when a new minor database engine version is available. Minor version upgrades often patch security vulnerabilities and fix bugs and therefore should be applied.",
-    "RelatedUrl": "https://aws.amazon.com/blogs/database/best-practices-for-upgrading-amazon-rds-to-major-and-minor-versions-of-postgresql/",
-    "Remediation": {
-        "Code": {
-            "NativeIaC": "https://docs.bridgecrew.io/docs/ensure-aws-db-instance-gets-all-minor-upgrades-automatically#cloudformation",
-            "Terraform": "https://docs.bridgecrew.io/docs/ensure-aws-db-instance-gets-all-minor-upgrades-automatically#terraform",
-            "CLI": "aws rds modify-db-instance --db-instance-identifier <db_instance_id> --auto-minor-version-upgrade --apply-immediately",
-            "Other": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/RDS/rds-auto-minor-version-upgrade.html"
-        },
-        "Recommendation": {
-            "Text": "Enable auto minor version upgrade for all databases and environments.",
-            "Url": "https://aws.amazon.com/blogs/database/best-practices-for-upgrading-amazon-rds-to-major-and-minor-versions-of-postgresql/"
-        }
-    },
-    "Categories": [],
-    "Notes": "",
-    "Compliance": {}
-}]
-```
+- GCP
+    | V3 | V4 |
+    |---|---|
+    | project_id | account_uid |
+    | location | region |
+    | resource_id | resource_name |
+    | resource_arn | resource_uid |
 
 
 ### JSON-OCSF
```

Comment:
Please update the finding example in the JSON-OCSF format.

Reply:
Done